pkg/agent/conversation.go

```diff
@@ -638,6 +684,50 @@ func (c *Agent) handleMetaQuery(ctx context.Context, query string) (answer strin
 		return "Available models:\n\n  - " + strings.Join(models, "\n  - ") + "\n\n", true, nil
 	case "tools":
 		return "Available tools:\n\n  - " + strings.Join(c.Tools.Names(), "\n  - ") + "\n\n", true, nil
+	case "session":
+		if c.session.ChatMessageStore != nil {
```

Comment:
This conditional will go away. Infact we can implement `.String()` method at the session level to return print friendly format.